### ISSUE TITLE
Extended info OSD option does not display when resuming movie from a widget

### ISSUE THREAD

**Issue by pheare** [croneter/PlexKodiConnect#337]
When resuming a movie from a widget, the 'extended information' option is greyed out on the osd.

Precondition:
1. Use a Skin that has the 'Extended Info' option in the  OSD when a movie is playing.  Horizon, for example.

Steps to reproduce:
1.  Load Horizon skin
2.  Create a widget based on movies - say Skin Helper - Recently Added
3.  Start movie from widget.  Bring up OSD - notice Extended Information is available
4.  Stop movie.
5.  Resume movie from widget created above.
6.  Bring up OSD

Expected result:
1. Extended Information is available.

Actual Result:
1. Extended information is not available.

Context:

**foghat Wrote:**
Think I found a bug: if you resume a movie from a widget, the Extended Info is greyed out in the OSD. I have tried from the script helper 'recently added' and 'now playing' widgets
If you resume from the 'all movies' list, Extended Info is selectable.

**Skin Developer Wrote:**
Works fine for me. Sounds like whatever is supplying your content is not setting the videoplayer content type correctly. The button will only show if content type of videoplayer is movies or episodes.

**Comment by pheare** [comment]
Hi @croneter not sure if you say Jurialmonkey's response to your question about this in the kodi forum?  Just in case, here it is:

Croneter Wrote:  
Hi there

Could you please elaborate on that? What attribute do I need to set for the ListItem? I cannot find anything specific to distinguish "video" between episodes and movies, e.g. in the doc http://mirrors.kodi.tv/docs/python-docs/...l#ListItem

Thanks!

It isn't a list item property. It is a property you have to set for the container. My python is pretty bad, but it is my understanding that it is this:
http://mirrors.kodi.tv/docs/python-docs/...setContent

**Comment by croneter** [comment]
Is this still an issue?

**Comment by pheare** [comment]
Hi, yes it was still behaving as described above with v1.8.18.

I just tried beta v2.0 and things are worse - Extended info on the OSD is now always grayed out regardless of how I play a movie (initial play from a widget, initial play from 'all movies', resume from widget, resume from 'all movies', etc).

I rolled back to 1.8.18 and rebuilt database and again, I see the behavior described in first post above.
